holidays/countries/pakistan.py

```diff
@@ -59,6 +59,10 @@ def _populate_public_holidays(self):
             # Labor Day.
             self._add_labor_day(tr("Labour Day"))
 
+        if self._year >= 2024:
+            # Youm-e-Takbeer.
+            self._add_holiday(tr("Youm-e-Takbeer"), (MAY, 28))
```

Comment:
```suggestion
            self._add_holiday_may_28(tr("Youm-e-Takbeer"))
```
Is there any source that this holiday has been established since 2024?

Reply:
The public and optional holidays webpage from the Cabinet Decision of the Government of Pakistan
https://cabinet.gov.pk/Detail/OTE2ODBiYmItZmI0MS00NDAwLWE5NGUtYmE1MGVjYzllMzAz

Document No.10-01/2024-Min-II - Declaration of Public Holiday on the Occasion of Youm-e-Takbeer
https://cabinet.gov.pk/SiteImage/Misc/files/Holidays/28-5-24.pdf

Reply:
Thank you for useful links!